magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py

```diff
@@ -500,24 +553,27 @@ def event_coincidence(input_file_lst, input_dir_magic, output_dir, config):
     )
 
     # Create the subarray description with the telescope coordinates
-    # relative to the center of the LST-1 and MAGIC positions
-    tel_descriptions = {
-        1: subarray_lst.tel[1],  # LST-1
-        2: subarray_magic.tel[2],  # MAGIC-I
-        3: subarray_magic.tel[3],  # MAGIC-II
-    }
-
-    subarray_lst1_magic = SubarrayDescription(
-        "LST1-MAGIC-Array", TEL_POSITIONS, tel_descriptions
+    # relative to the center of the LST and MAGIC positions
+    tel_descriptions = {}
+    for k, v in TEL_NAMES.items():
+        if v[:3] == "LST":
+            tel_descriptions[k] = subarray_lst.tel[k]
+        else:
```

Comment:
elif v[:5] == "MAGIC"
...
else
<throw exception>

Reply:
also taking telescopes and indixes from two different subarrays looks prone to errors (I'm not sure if it will work fine if people start shuffling the indicies here)

Reply:
Sorry, maybe a stupid Python question but...this way do we check, in the elif, if the first 6 telescope names are MAGIC? Should we use this to raise an exception?

Reply:
I've meant that you have 
if ... MAGIC...

... MAGIC stuff...
elif ...LST .....
... LST stuff...
else
... here throw exception because behavior is not implemented for other types of telescopes...